mk/uvmt/vsim.mk

```diff
@@ -682,7 +682,7 @@ cov: $(COV_MERGE_TARGET)
 # Clean up your mess!
 
 clean:
-	rm -rf $(SIM_RUN_RESULTS)
+	rm -rf $(SIM_RESULTS)
```

Comment:
Here if this ->SIM_RESULTS if used, it would mean the entire directory vsim_results/xrun_result/vcs_result will be removed. 
So need to be careful using clean after this for older users of cv32e40p.
Previously SIM_RUN_RESULTS would only clean the current test's run directory.

Maybe we will add a new target like "clean_test_dir" to only test run dir.

Reply:
As there is already the `clean_all` target, maybe add that `rm -rf $(SIM_RESULTS)` under that ?